src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,157 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> headNode;
+    private Node<T> tailNode;
+
+    public MyLinkedList() {
+        this.size = 0;
```

Comment:
if you want you can remove initialization from the constructor because `int` value by default has 0 value.